plugins/modules/nd_rest.py

```diff
@@ -48,6 +48,16 @@
     - The file path containing the body of the HTTP request.
     type: path
     aliases: [ config_file ]
+  suppress_previous:
+    description:
+    - If enabled, a GET call to check previous object state will not be sent before a PUT, PATCH and DELETE update to ND.
+    - This causes the previous return value to be empty.
+    - The previous state of the object will not be checked and PUT, PATCH and DELETE update calls to ND will contain all properties specified in the task.
+    - Certain ND API endpoints do not support the GET method for querying the current status of an object before updating or deleting it.
+    - Use O(suppress_previous=true) to avoid making unsupported GET requests to such ND API endpoints.
+    type: bool
+    aliases: [ no_previous, ignore_previous ]
```

Comment:
In cisco.aci we also default this to false to clear up any ambiguity as None/false are the same.
```suggestion
    aliases: [ no_previous, ignore_previous ]
    default: false
```